### PR TITLE
Fix issue with merge when there is not found exception

### PR DESCRIPTION
**Pull request by cjmyers**

### CODE REVIEW

File: java/src/main/java/org/synbiohub/PrepareSubmissionJob.java [hunk 1]
```diff
@@ -482,8 +484,14 @@ public void visit(Identified identified, TopLevel topLevel) {
 						SBOLDocument tlDoc;
 						try {
 							tlDoc = sbh.getSBOL(URI.create(topLevelUri));
-						} catch (SynBioHubException e) {
-							tlDoc = null;
+						} catch (SynBioHubException e1) {
+							StringWriter sw = new StringWriter();
```
Comment (3ach) [inline java/src/main/java/org/synbiohub/PrepareSubmissionJob.java]: You can't `printStackTrace(System.err)` ?

File: java/src/main/java/org/synbiohub/PrepareSubmissionJob.java [hunk 2]
```diff
@@ -482,8 +484,14 @@ public void visit(Identified identified, TopLevel topLevel) {
 						SBOLDocument tlDoc;
 						try {
 							tlDoc = sbh.getSBOL(URI.create(topLevelUri));
-						} catch (SynBioHubException e) {
-							tlDoc = null;
+						} catch (SynBioHubException e1) {
```
Comment (3ach) [inline java/src/main/java/org/synbiohub/PrepareSubmissionJob.java]: Won't this still fail in the event of a NotFoundException?

Comment (cjmyers) [reply java/src/main/java/org/synbiohub/PrepareSubmissionJob.java]: I changed libSBOLj to return null on NotFoundException.

File: java/src/main/java/org/synbiohub/PrepareSubmissionJob.java (hunk 1 above)
Comment (cjmyers) [reply java/src/main/java/org/synbiohub/PrepareSubmissionJob.java]: Probably, but I was also converting to a string to return in errorLog.

**Pull request merged**